src/App.jsx

```diff
@@ -0,0 +1,37 @@
+import "./App.css";
+import AddTodo from "./Components/Elements/AddTodo/addTodo";
+import EditTodo from "./Components/Elements/EditTodo/editTodo";
+import MainContent from "./Components/Home/mainContent";
+import Sidebar from "./Components/Home/sidebar";
+import Todo from "./Components/TodoPage/Todo";
+import { GlobalProvider } from "./context/GlobalState";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+
+function App() {
+	return (
+		<GlobalProvider>
+			<Router>
+				<Routes>
+					<Route
+						path='/'
+						element={
+							<div className='App'>
+								<Sidebar />
+								<MainContent />
```

Comment:
Use some kind of code formatter i.e. prettier

Reply:
Its not working, but will try again